src/Compilers/CSharp/Test/Semantic/Semantics/LambdaTests.cs

```diff
@@ -31,7 +31,11 @@ class C
             comp.VerifyDiagnostics(
                 // (7,52): error CS1586: Array creation must have array size or array initializer
                 //     Expression<Action<dynamic>> e = x => new object[](x);
-                Diagnostic(ErrorCode.ERR_MissingArraySize, "[]"));
+                Diagnostic(ErrorCode.ERR_MissingArraySize, "[]").WithLocation(7, 52),
+                // (7,42): error CS0149: Method name expected
```

Comment:
>error CS0149: Method name expected [](start = 27, length = 34)

This cascading error seems unfortunate. The first error makes sense but this second error seems incorrect given the first.